widget/embedded/src/components/WidgetEvents.tsx

```diff
@@ -42,10 +42,16 @@ export function WidgetEvents() {
         toAccount && getWalletsDetails([toAccount]);
       }
       if (
-        event.type === StepEventType.TX_EXECUTION_BLOCKED &&
-        validBlockedStatuses.includes(event.status)
+        (event.type === StepEventType.TX_EXECUTION_BLOCKED &&
+          validBlockedStatuses.includes(event.status)) ||
+        event.type === StepEventType.FAILED
```

Comment:
I didn't understand your intention about this line?
`event.type === StepEventType.FAILED
`

Reply:
As I understand it, we don't trigger RouteEvent all the time, so I put it to make sure the notifier is called.

Reply:
Please create another task to handle it.